TMDbLib/Objects/TvShows/TvShow.cs

```diff
@@ -75,7 +75,7 @@ public class TvShow
         public string Name { get; set; }
 
         [JsonProperty("networks")]
-        public List<Network> Networks { get; set; }
+        public List<NetworkBase> Networks { get; set; }
```

Comment:
Don't we need a custom converter for this?

Deserializing to concrete types is a messy affair.